src/NuGet.Core/NuGet.Protocol/PublicAPI/net472/PublicAPI.Shipped.txt

```diff
@@ -1494,7 +1494,7 @@ NuGet.Repositories.NuGetv3LocalRepository.Exists(string packageId, NuGet.Version
 NuGet.Repositories.NuGetv3LocalRepository.FindPackage(string packageId, NuGet.Versioning.NuGetVersion version) -> NuGet.Repositories.LocalPackageInfo
 NuGet.Repositories.NuGetv3LocalRepository.FindPackagesById(string packageId) -> System.Collections.Generic.IEnumerable<NuGet.Repositories.LocalPackageInfo>
 NuGet.Repositories.NuGetv3LocalRepository.NuGetv3LocalRepository(string path) -> void
-NuGet.Repositories.NuGetv3LocalRepository.NuGetv3LocalRepository(string path, NuGet.Protocol.LocalPackageFileCache packageFileCache, bool isFallbackFolder) -> void
```

Comment:
Please don't break public APIs. Create an overload and call the new one from the old one, passing in default values for the new parameters.

Reply:
Ok